packages/cli/test/simulation/singleNodeSingleValidatorMultipleEpoch.test.ts

```diff
@@ -0,0 +1,51 @@
+import {FAR_FUTURE_EPOCH, logFilesDir, SimulationEnvironment} from "../utils/simulation/index.js";
+import {missedBlocksAssertions, participationAssertions} from "../utils/simulation/assertions.js";
+
+const epochLimit = 5;
+
+describe("singleNodeSingleValidatorMultipleEpoch", function () {
```

Comment:
Is this test redundant to singleNodeForks? What is this test covering that singleNodeForks is not? Time is CI is precious for productivity, tests should add value comparable to the time they take

Reply:
It's covering `missedBlocksAssertions` and `participationAssertions`, the `singleNodeForks` only covers the forks. As mentioned earlier I wanted it to be a scope specific test files, but it seems you want to merge them together. Will do that.